### PR TITLE
Twitter Demo Link Not Working 

### PR DESCRIPTION
**Pull request by fixelrr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,8 +205,7 @@ _(scroll right on table to see all 5 columns)_
...
+| Twitter                        | [twitter-geek.netlify.app](https://twitter-geek.netlify.app/#/) <br/> [github.com (screenshot)](https://github.com/RisingGeek/twitter-clone/blob/master/Live/feed.png) | [GitHub](https://github.com/RisingGeek/twitter-clone) <br/> [GitHub](https://github.com/RisingGeek/twitter-clone-backend)                  | React, Redux, NodeJS, MYSQL                                                    | ![GitHub Repo stars](https://flat.badgen.net/github/stars/RisingGeek/twitter-clone) ![GitHub Repo stars](https://flat.badgen.net/github/stars/RisingGeek/twitter-clone-backend)
```
Comment (iamrishupatel) [inline README.md:208]: Can you please remove this entry as well?

**Review by iamrishupatel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (iamrishupatel) [reply README.md:208]: https://twitter-geek.netlify.app/#/
This link is not working as well.

**Comment by iamrishupatel** [comment]
@fixelrr Could you make the required changes and resolve the merge conflict

**Comment by Pritamgit2002** [comment]
Provide demo link

**Pull request closed without merging**